src/app/admin/email-templates/email-template-form.component.ts

```diff
@@ -79,21 +92,62 @@ export class AdminEmailTemplateFormComponent extends FormComponent<EmailTemplate
     return this._ckEditorConfig;
   }
 
+  @ViewChild(CKEditorComponent, { static: false })
+  public set ckEditor(value: CKEditorComponent) {
+    this._ckEditor = value;
+  }
+
   constructor(service: AdminEmailTemplateService,
               router: Router,
               route: ActivatedRoute,
               notificationService: NotificationService,
               breadcrumbService: BreadcrumbService,
-              protected _translateService: TranslateService) {
+              protected _translateService: TranslateService,
+              protected _cdr: ChangeDetectorRef) {
     super(service, router, route, notificationService, breadcrumbService);
   }
 
   public ngOnInit() {
     this._ckEditorConfig.language = this._translateService.currentLang;
+
+    this._onLanguageChange();
+
     super.ngOnInit();
   }
 
+  public ngOnDestroy() {
+    this._destroy$.next(true);
+    this._destroy$.complete();
+  }
+
   public addText(editorArea: CKEditorComponent, text: string) {
     editorArea.instance.insertText(`{{ ' ${text} ' }}`);
   }
+
+  protected _removeCurrentEditor(): void {
+    this._ckEditor.instance.removeAllListeners();
+
+    window['CKEDITOR'].instances[this._ckEditor.instance.name].destroy();
+    document.getElementsByClassName(this._ckEditor.instance.id).item(0).remove();
+
+    this._ckEditor.instance.destroy();
+    this._ckEditor.instance = null;
+  }
+
+  protected _onLanguageChange(): void {
```

Comment:
The name of the method need to be changed to reflect its logic. Currently the name says that the method will bee called on language change. It need to be renamed to something like `_initLanguageChangeWatch`.